pkg/spec/spec.go

```diff
@@ -37,7 +37,7 @@ type ServicePortMod struct {
 type ServiceSpecMod struct {
 	api_v1.ServiceSpec `json:",inline"`
 	Name               string           `json:"name,omitempty"`
-	Ports              []ServicePortMod `json:"ports"`
+	Ports              []ServicePortMod `json:"ports,conflicting"`
```

Comment:
No, we need to add this JSON tag to every conflicting tag that we have handled.
e.g. Ports are conflicting in the sense that they exist in SeviceSpec as well as in our spec in ServiceSpecMod, so we need to declare it as conflicting meaning that this is a known conflicting tag and it has been handled.